packages/angular_devkit/build_angular/src/webpack/plugins/common-js-usage-warn-plugin.ts

```diff
@@ -100,13 +103,17 @@ export class CommonJsUsageWarnPlugin {
     });
   }
 
-  private hasCommonJsDependencies(dependencies: WebpackModule[], checkParentModules = false): boolean {
+  private hasCommonJsDependencies(
+    compilation: compilation.Compilation,
+    dependencies: WebpackModule[],
+    checkParentModules = false): boolean {
     for (const dep of dependencies) {
       if (dep instanceof CommonJsRequireDependency || dep instanceof AMDDefineDependency) {
         return true;
       }
 
-      if (checkParentModules && dep.module && this.hasCommonJsDependencies(dep.module.dependencies)) {
+      const module = getWebpackModule(compilation, dep);
```

Comment:
Can we avoid this when `checkParentModules` is false?

Reply:
ok